src/Components/Aspire.Azure.Storage.Blobs/AzureBlobStorageContainerSettings.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using System.Globalization;
+using System.Text;
+using Aspire.Azure.Common;
+
+namespace Aspire.Azure.Storage.Blobs;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to Azure Blob Storage container.
+/// </summary>
+public sealed class AzureBlobStorageContainerSettings : AzureStorageBlobsSettings, IConnectionStringSettings
+{
+    public string? BlobContainerName { get; set; }
+
+    void IConnectionStringSettings.ParseConnectionString(string? connectionString)
+    {
+        if (string.IsNullOrEmpty(connectionString))
+        {
+            return;
+        }
+
+        if (Uri.TryCreate(connectionString, UriKind.Absolute, out var uri))
+        {
+            ServiceUri = uri;
+
+            // TODO: how do we get the container name from the URI?
```

Comment:
I'm not quite sure how to proceed here. How do I bootstrap a test scenario to hit this code path?

Reply:
Does the container name ever come on the Uri?